projects/odyssey/index.js

```
@@ -38,18 +50,42 @@ Object.keys(graphs).forEach(chain => {
         tvl: async (api) => {
             if ( Date.now() / 1000 - api.timestamp > 60 * 60 * 24 ) {
                 // do historical query
-                let depositedUSD = {}
-                const { positionRegistryDailyDatas } = await cachedGraphQuery(`odyssey-${chain}-historical`, graphs[chain], queries.previous, { variables: { time_start: api.timestamp - 60 * 60 * 24, time_end: api.timestamp } })
-                positionRegistryDailyDatas.map(({ totalDepositedUSD, id, dayStartTimestamp }) => {
+                let depositedBalances = {}
+                const { positionRegistrySnapshots } = await cachedGraphQuery(`odyssey-${chain}-historical`, graphs[chain], queries.previous, { variables: { time_start: api.timestamp - 60 * 60 * 24, time_end: api.timestamp } })
+                positionRegistrySnapshots.map(({ smartAccountSnapshots, id, dayStartTimestamp }) => {
```

Biome (2.3.13)

[error] This callback passed to map() iterable method should always return a value.

Add a return with a value to this callback.

(lint/suspicious/useIterableCallbackReturn)